### PR TITLE
Gradle 7.6.3

### PR DESCRIPTION
**Pull request by Goooler**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -145,11 +145,6 @@ subprojects {
             }
         }
 
-        compileGroovy {
-            sourceCompatibility = project.sourceCompatibility
-            targetCompatibility = project.targetCompatibility
-        }
```
Comment (Goooler) [inline build.gradle:151]: This could be removed now since #703 gets merged.

**Comment by aalmiray** [comment]
@ysb33r let's keep an eye on the build for this PR to merge it before `4.0.1`

File: slides-export/src/main/groovy/org/asciidoctor/gradle/slides/export/base/AbstractExportBaseTask.groovy [hunk 2]
```diff
@@ -280,10 +277,8 @@ abstract class AbstractExportBaseTask extends DefaultTask {
         dependsOn tasks.findAll {
             if (it instanceof Task) {
                 true
-            } else if (GRADLE_GE_4_8) {
-                it instanceof org.gradle.api.tasks.TaskProvider
             } else {
-                false
+                it instanceof org.gradle.api.tasks.TaskProvider
```
Comment (ysb33r) [inline slides-export/src/main/groovy/org/asciidoctor/gradle/slides/export/base/AbstractExportBaseTask.groovy:281]: We can now replace this if-else with `findAll { it instanceof Task || it instanceof TaskProvider }`.

No need for the `CompileDynamic` either as it was only required for compatibility with Gradle 4.7 and older.

**Review by ysb33r — CHANGES_REQUESTED** [review]

File: slides-export/src/main/groovy/org/asciidoctor/gradle/slides/export/base/AbstractExportBaseTask.groovy (hunk 2 above)
Comment (Goooler) [reply slides-export/src/main/groovy/org/asciidoctor/gradle/slides/export/base/AbstractExportBaseTask.groovy]: c0486193b8616dc366eaa3ea1e55cc6729288ccc

**Comment by Goooler** [comment]
There are still some issues with updating Gradle 8.5 and JDK 17 at https://github.com/Goooler/asciidoctor-gradle-plugin/pull/62, I'll send it after fixing that.

**Comment by ysb33r** [comment]
Thanks. There are some nice cleanups in there.

**Pull request merged**